Permanent/ViewControllers/FileDetailsViewController.swift

```diff
@@ -265,10 +266,18 @@ extension FileDetailsViewController: UICollectionViewDataSource {
             }
             returnedCell = cell
         case .tags:
-            let cell = collectionView.dequeueReusableCell(withReuseIdentifier: FileDetailsBottomCollectionViewCell.identifier, for: indexPath) as! FileDetailsBottomCollectionViewCell
-            cell.configure(title: title(forCellType: currentCellType), details: stringCellDetails(cellType: currentCellType))
-
-            returnedCell = cell
+            if stringCellDetails(cellType: currentCellType) == ""
+            {
+                let cell = collectionView.dequeueReusableCell(withReuseIdentifier: FileDetailsBottomCollectionViewCell.identifier, for: indexPath) as! FileDetailsBottomCollectionViewCell
+                cell.configure(title: title(forCellType: currentCellType), details: "Tap to add tags".localized())
+                returnedCell = cell
+            } else {
+                let cell = collectionView.dequeueReusableCell(withReuseIdentifier: TagsNamesCollectionViewCell.identifier, for: indexPath) as! TagsNamesCollectionViewCell
```

Comment:
Let's stick with a single collection view cell for both cases here.
TagsNamesCollectionViewCell should know to present a "Tap to add tags" text if there are no tags available

Reply:
TagsNamesCollectionViewCell is presenting both cases.